requirements/requirements.txt

```diff
@@ -1,5 +1,4 @@
 colorama
-distrax @ git+https://github.com/google-deepmind/distrax  # distrax release doesn't support jax > 0.4.13
```

Comment:
As discussed offline please keep this in for now :pray:

Reply:
WIll do!